ddl/ddl_worker.go

```diff
@@ -295,7 +208,7 @@ func (d *ddl) addBatchDDLJobs(tasks []*limitJobTask) {
 	startTime := time.Now()
 	var err error
 	// DDLForce2Queue is a flag to tell DDL worker to always push the job to the DDL queue.
-	toTable := variable.EnableConcurrentDDL.Load() && !variable.DDLForce2Queue.Load()
+	toTable := !variable.DDLForce2Queue.Load()
```

Comment:
Is the variable `DDLForce2Queue` still useful?

Reply:
May be used when upgrading from old cluster.

Reply:
Can't we make the assumption like "when the cluster is upgrading to this version, the concurrent DDL must be enabled"?